src/network/networkd-address.c

```diff
@@ -2096,3 +2153,67 @@ int network_drop_invalid_addresses(Network *network) {
 
         return 0;
 }
+
+int config_parse_address_ipv4_nft_set_context(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        Network *network = userdata;
+        _cleanup_(address_free_or_set_invalidp) Address *n = NULL;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(network);
+
+        r = address_new_static(network, filename, section_line, &n);
+        if (r == -ENOMEM)
+                return log_oom();
+        if (r < 0) {
+                log_syntax(unit, LOG_WARNING, filename, line, r,
+                           "Failed to allocate a new address, ignoring assignment: %m");
+                return 0;
+        }
+
+        return config_parse_nft_set_context(unit, filename, line, section, section_line, lvalue, ltype, rvalue, &n->ipv4_nft_set_context, &n->n_ipv4_nft_set_contexts);
```

Comment:
> It looks like directive `Address=10.5.0.1/24` is not applied:

This is because no `TAKE_PTR(n)` after `config_parse_nft_set_context()`, hence the Address object will be freed.

Reply:
Thanks! Fixed.